### PR TITLE
Hide DatePicker while not in use.

### PR DESCRIPTION
**Pull request by ashishkakkad8**
fixes #9

### CODE REVIEW

**Comment by aaryankotharii** [comment]
Hey @ashishkakkad8 Thank you for interest in this issue. Great work with hiding the datePicker!
I'd like to point of a minor bug, When the user taps on`endDateTextField`, the datepicker does hide but the keyboard shows up and hides its contents. How about we set the inputView of `endDateTextField` as `endDatePicker`.
know more about input views [here](https://developer.apple.com/documentation/uikit/uiresponder/1621092-inputview)

File: Application/To-Do/Controller/TaskDetailsViewController.swift [hunk 1]
```diff
@@ -76,3 +78,18 @@ class TaskDetailsViewController: UIViewController {
...
+extension TaskDetailsViewController : UITextFieldDelegate {
+    func textFieldDidEndEditing(_ textField: UITextField) {
+        if textField == endDateTextField {
+            endDatePicker.isHidden = true
+        }
+    }
+
+    func textFieldDidBeginEditing(_ textField: UITextField) {
+        if textField == endDateTextField {
+            endDatePicker.isHidden = false
```
Comment (aaryankotharii) [inline Application/To-Do/Controller/TaskDetailsViewController.swift:92]: since we already have the delegate methods, we can set the inputview here!

**Review by aaryankotharii — CHANGES_REQUESTED** [review]

**Comment by ashishkakkad8** [comment]
I have updated as you asked for input views.

**Comment by aaryankotharii** [comment]
perfect! Now that `endDatePicker ` is a variable please move it under the comment for // variables. Lastly do a self review to make sure everything is working as intended.

**Comment by aaryankotharii** [comment]
Great! I am approving this pr.

**Review by aaryankotharii — APPROVED** [review]

**Pull request merged**